sdk/communication/AzureCommunicationChat/Source/PushNotification/PushNotificationClient.swift

```diff
@@ -61,7 +61,7 @@ internal class PushNotificationClient {
 
     internal func startPushNotifications(
         deviceRegistrationToken: String,
-        encryptionKeyPair: EncryptionKeyPair,
+        pushNotificationEncryptionDelegate: PushNotificationEncryptionDelegate?,
```

Comment:
While this is internal, I would recommend following the same pattern as the public API and not setting the delegate in a method like this.

Reply:
I set the delegate in chatClient and it will not be passed into pushNotificationClient.